packages/fx-core/resource/package.nls.json

```diff
@@ -255,7 +255,7 @@
   "core.TabSPFxOption.detailNew": "Build UI with SharePoint Framework",
   "core.TabNonSso.label": "Basic Tab",
   "core.TabNonSso.detail": "A simple implementation of a web app that's ready to customize",
-  "core.copilotPlugin.api.noAuth": "None auth",
+  "core.copilotPlugin.api.noAuth": "No authentication",
   "core.copilotPlugin.api.apiKeyAuth": "API key auth(Bearer token auth)",
```

Comment:
I am wondering whether this need to be API Key authentication. :)

Reply:
@supkasar , do you have any suggestions about the API Key auth?

Reply:
If you think, users won't understand what authentication we are talking about here, then 'No API key auth' makes sense. But if they would understand it easily, then the current content is enough.